system/settings/settings.xml

```diff
@@ -91,7 +91,7 @@
           <control type="list" format="string" />
         </setting>
       </group>
-      <group id="2">
+      <group id="2" label="371" help="-1">
```

Comment:
don't get me wrong - the ability to assign a help string in general is nice (and also needed for your ADSP stuff IIRC), but defining this attribute here when it's not used in this case didn't make sense.